ci_framework/roles/edpm_build_images/defaults/main.yml

```diff
@@ -0,0 +1,49 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# All variables intended for modification should be placed in this file.
+# All variables within this role should have a prefix of "cifmw_edpm_build_images"
+
+cifmw_edpm_build_images_basedir: "{{ cifmw_basedir | default(ansible_user_dir ~ '/ci-framework') }}"
+cifmw_edpm_build_images_src: "https://github.com/openstack-k8s-operators/edpm-image-builder"
+cifmw_edpm_build_images_artifacts: "{{ cifmw_edpm_build_images_basedir }}/artifacts/edpm_images"
+cifmw_edpm_build_images_packages:
+  - python3-devel
+  - python3-pip
+  - podman
+  - buildah
+  - qemu-img
+  - binutils
+  - bzip2
+  - util-linux
+  - dosfstools
+  - gdisk
+  - kpartx
+  - sudo
+  - curl
+  - git
+  - lvm2
+cifmw_edpm_build_images_config_dir: "{{ cifmw_edpm_build_images_basedir }}/tmp/edpm-image-builder/images"
+cifmw_edpm_build_images_elements:
+  - "{{ cifmw_edpm_build_images_basedir }}/venv/share/edpm-image-builder/dib"
+  - "{{ cifmw_edpm_build_images_basedir }}/venv/share/ironic-python-agent-builder/dib"
+cifmw_edpm_build_images_all: false
+cifmw_edpm_build_images_hardened_uefi: true
+cifmw_edpm_build_images_ironic_python_agent: true
+cifmw_edpm_build_images_hardened_uefi_package: true
```

Comment:
I would say keep all these values to false and set these values to true in scenarios/centos-9/edpm_build_images_ci.yml file. It will help users to manipulate these vars.